docs/tutorial/quickstart.md

```diff
@@ -0,0 +1,253 @@
+# Tutorial: Get started with oras-go v2
+
+This tutorial introduces the basics of managing OCI artifacts with the [oras-go v2](https://pkg.go.dev/oras.land/oras-go/v2) package.
+
+You'll get the most out of this tutorial if you have a basic familiarity with Go and its tooling. If this is your first exposure to Go, please see [Tutorial: Get started with Go](https://golang.org/doc/tutorial/getting-started) for a quick introduction.
+
+The tutorial includes the following sections:
+
+1. Connect to a remote repository.
+2. Show tags in the repository.
+3. Push a layer to the repository.
+4. Push a manifest to the repository.
+5. Fetch the manifest from the repository.
+6. Copy the artifact from the repository.
+
+The complete code is provided at the end of this tutorial.
+
+## Connect to a remote repository with basic authentication
```

Comment:
Let's have a section beforehand that illustrates how to import the package

Reply:
added